src/locale/languages/en.js

```diff
@@ -157,6 +157,9 @@ export const en = {
     followingList: {
       title: 'Following',
     },
+    followYou: {
+      title: 'Follows you',
+    },
```

Comment:
I just realised that we have France translation. I will add the `followYou` key there